package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "s3utils",
-  "version": "1.12.0",
+  "version": "1.11.11",
```

Comment:
?

Reply:
The 1.11 branch has a wrong version in package.json (I also saw that :laughing: )

Reply:
as discussed on slack, we should just forget 1.11, and use 1.12 in Artesca 1.3